tests/Feature/ReturnRegisterDataTest.php

```diff
@@ -93,11 +92,42 @@ public function updateTest()
             'email' => 'qwerty112@gmail.com',
         ];
         $this->actingAs($user, 'api');
-        $response = $this->put('/api/auth/users/40', $data);
+        $response = $this->put('/api/auth/update/'.$user->id, $data);
         $response->assertStatus(200);
         $response->assertJsonStructure(['token']);
         $user->refresh();
         $this->assertEquals($user->name, $data['name']);
         $this->assertEquals($user->email, $data['email']);
     }
+    /** @test */
+    public function getUsersTest()
+    {
+        $count = 5;
+        $users = User::factory()->count($count)->create([]);
+        $user = $users->first();
+        $this->actingAs($user, 'api');
+        $response = $this->get('/api/auth/users');
+        $response->assertStatus(200);
+        $response->assertJsonStructure(['users']);
+        $this->assertCount($count, $users);
+    }
+    public function getUserDataTest()
+    {
+        $user = User::factory()->create([
+            'name' => 'Qwertyqweqwe',
+            'email' => 'qwerty2@gmail.com',
+        ]);
+        $anotherUser = User::factory()->create([
+            'name' => 'Qwertyqw33321we',
+            'email' => 'q1112werty2@gmail.com',
+        ]);
+        $this->actingAs($user, 'api');
+        $response = $this->get('/api/auth/users/'.$user->id);
+        $response->assertStatus(200);
+        $response->assertJsonStructure(['users']);
+
+        $this->actingAs($anotherUser, 'api');
```

Comment:
looks good, but it should be 2 separate tests. 1 for success case and one for 403 case